lib/__tests__/ignoreDisables.test.js

```diff
@@ -61,10 +61,10 @@ describe('ignoreDisables with standalone', () => {
 	});
 
 	it('expected rule 0', () => {
-		expect(results[0].warnings[0].text.indexOf('block-no-empty')).not.toBe(1);
+		expect(results[0].warnings[0].text.includes('block-no-empty')).not.toBe(1);
 	});
 
 	it('expected rule 1', () => {
-		expect(results[0].warnings[1].text.indexOf('block-no-empty')).not.toBe(1);
+		expect(results[0].warnings[1].text.includes('block-no-empty')).not.toBe(1);
```

Comment:
Looks like wrong comparison.

Reply:
Yup, I will revert these two, thanks for catching this.

Reply:
BTW really nice to see eslint to switch `foo.includes(string).toBe(true)` to `foo.contains(string)` :)